src/group.rs

```diff
@@ -32,28 +35,46 @@ impl GroupCreateOpts {
     /// - `add_as_member`
     ///     - `true` - The creating user should be added as a member (in addition to being a group admin)
     ///     - `false` - The creating user will not be a member of the group, but will still be an admin.
+    /// - `members` - List of users to be added as members of the group.
+    ///               Note: even if `add_as_member` is false, the calling user will be added as a member if they are in this list.
     /// - `needs_rotation`
     ///     - true - group's private key will be marked for rotation
     ///     - false (default) - group's private key will not be marked for rotation
     pub fn new(
         id: Option<GroupId>,
         name: Option<GroupName>,
         add_as_member: bool,
+        members: Vec<UserId>,
         needs_rotation: bool,
     ) -> GroupCreateOpts {
         GroupCreateOpts {
             id,
             name,
             add_as_member,
+            members,
             needs_rotation,
         }
     }
+
+    fn standardize(self, calling_id: &UserId) -> GroupCreateOpts {
+        let mut new_members = self.members.clone();
```

Comment:
You might be able to do this clone inside the  if to make it not need to happen in the case where we don't have to add the calling users id.

Reply:
I made this a bit uglier but hopefully also better